framework/src/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -116,17 +136,17 @@
      * @param path
      *            The current tree path.
      */
-    public static FlowExpressions. /*@Nullable*/ Receiver parse(String s,
+    public static FlowExpressions. /*@Nullable*/ Receiver parseAllowingItself(String s,
```

Comment:
I'm fine with keeping it as is. I was just confused by the inconsistency of the long name e.g. on line 145 and the short name on 148.